### ISSUE TITLE
Subscription emails: Featured Image not embedded with Excerpt

### ISSUE THREAD

**Issue by formosattic** [Automattic/jetpack#32684]
### Impacted plugin

Jetpack

### Quick summary

When the post shared has an excerpt either set manually or with Read More, the featured image set or image in the content is not shown in the email.

Based on https://github.com/Automattic/jetpack/issues/2957 , the featured image should be embedded if one is set.

### Steps to reproduce

1. Under Settings > Reading, set "For each post in a feed, include" to `Excerpt`.
2. Create a post with a manual excerpt.
3. Add a featured image in the settings
4. Publish

### A clear and concise description of what you expected to happen.

Featured image should show.

### What actually happened

No featured image.

| Editor | Email |
| ------------- | ------------- |
| ![image](https://github.com/Automattic/jetpack/assets/15944096/8fd2cf9e-afb1-44b0-afba-e87331a1f440)  | ![image](https://github.com/Automattic/jetpack/assets/15944096/a22c09e9-fbfa-4bc7-9999-2b5d57313fc2)  |

### Impact

One user reported so far, and I was able to reproduce.

User report: 6551798-zd-a8c

### Available workarounds?

Not aware of a workaround but the platform is still usable

### Logs or notes

Only tested for self-hosted.

**Comment by jeherve** [comment]
@mashikag Does this ring a bell? Do you know if the featured image is meant to be displayed when only the excerpt should be displayed in a subscription email?

**Comment by TimBroddin** [comment]
I'm unable to replicate this

![CleanShot 2023-09-25 at 09 47 05@2x](https://github.com/Automattic/jetpack/assets/528287/c4cd7e7a-1d68-45dc-ae0b-da086a9c25dc)

@mashikag did you fix this already?

**Comment by mashikag** [comment]
> I'm unable to replicate this

How did you set the featured image? Using a post block, or the post settings? Unfortunately, the two are two different concepts. The toggle in the newsletter settings relates to the featured image that is set in a post's settings.

> @mashikag did you fix this already? 

Nope, I did not touch this yet. 🙂 

> Do you know if the featured image is meant to be displayed when only the excerpt should be displayed in a subscription email?

I checked both the project thread (pdDOJh-Nu-p2) and the related CfT (pdtkmj-Gy-p2). I did not find any info whether the feature image should be displayed in a subscription email where post excerpt is included.

@yansern, what is your take on it? I think it would make sense to display the featured image for post excerpts too, when the feature image toggle is on.

**Comment by mashikag** [comment]
<table>
  <tr>
    <td><img src="https://github.com/Automattic/jetpack/assets/2019970/a2dd20f2-00c1-4034-b693-6f8d4fd3efbd" width="200"></td>
    <td><img src="https://github.com/Automattic/jetpack/assets/2019970/ef1d1702-be12-4784-8757-5affa2f7aa48" width="200"></td>
    <td><img src="https://github.com/Automattic/jetpack/assets/2019970/ccaadfaa-259c-49ff-bc17-8d7f44667c19" width="200"></td>
  </tr>
</table>

I tried to repro this myself, and similarly to @TimBroddin, I was unable to repro this. The image is being displayed when the newsletter is set to send post excerpts, and the feature image toggle is on. 🤔

**Comment by yansern** [comment]
> @yansern, what is your take on it? I think it would make sense to display the featured image for post excerpts too, when the feature image toggle is on.

From the product perspective, its nice to show it. :)

**Comment by formosattic** [comment]
Aha, thanks y'all!  @mashikag's screenshot of the settings was helpful. 

I missed the option "Enable featured image on your new post emails" under `Settings > Newsletter`.  I see that the user doesn't have it on either, so that must be it although unexpected for them because the featured image used to display before.

I'll close this for now.

**Comment by irene-markelic** [comment]
Hello, I do have this problem and it is costing me life time. I have "Enable featured image" under Jetpack->Settings->Newsletter checked and also "show excerpt". I have a featured image. And it doesn't show in the email-preview. It did before, but then I replaced the featured image and after that it disappeared. I have tried to repeat the same steps, e.g. set to "show full text" and then back to "show excerpt" again, hoping that this would trigger something in the background but no luck so far.
